### PR TITLE
Test for fr env / APP_SECRETS before running commands

### LINKED ISSUE #87: Print a warning when trying to run Craft Copy on the fortrabbit App
I speak with Craft Copy users in our support channel. There seems to be some confusion how and where Craft Copy works. We have now tried to highlight in the REAMDE of Craft Copy that Craft Copy should always be run locally.

But I believe some users will still try to run it on the App itself. Therefore I suggest here to catch that error and print out an error or maybe just a warning like:

```
Looks like you are trying to run Craft Copy on the fortrabbit. That's not how it usually works. Best run Craft Copy with your local development environment. More here: https://github.com/fortrabbit/craft-copy#workflow
```

### PR DESCRIPTION
**Pull request by tomdavies**
fixes #87

### CODE REVIEW

File: src/Actions/StageAwareBaseAction.php [hunk 1]
```diff
@@ -142,4 +148,9 @@ protected function getStageName(): ?string
             ?? getenv(Plugin::ENV_DEFAULT_STAGE)
                 ?: 'production';
     }
+
+    protected function isFortrabbitEnv(): bool
+    {
+        return getenv('APP_SECRETS') && getenv('APP_SECRETS') == '/etc/secrets.json';
```
Comment (ostark) [inline src/Actions/StageAwareBaseAction.php:154]: the first `getenv()` is not needed.

**Review by ostark — CHANGES_REQUESTED** [review]
getenv() returns something falsy if the var is not set.

**Comment by tomdavies** [comment]
@ostark done

**Comment by frank-laemmer** [comment]
https://github.com/fortrabbit/craft-copy/issues/87

**Comment by ostark** [comment]
Replaced by https://github.com/fortrabbit/craft-copy/pull/156 for no reason (copy&paste was easier than resolving git conflicts)

**Pull request closed without merging**